includes/Data/SiteFeatures.php

```diff
@@ -0,0 +1,128 @@
+					'desc'     => 'Get more traffic to your WordPress site with powerful analysis and tools from our colleagues at Yoast.',
+					'selected' => false,
+					'shown'    => true,
+				),
+				'creative-mail-by-constant-contact'  => array(
+					'slug'     => 'creative-mail-by-constant-contact',
+					'icon'     => '--site-features-email',
+					'title'    => 'Email Newsletters',
+					'subtitle' => 'Powered by Creative Email',
+					'desc'     => 'A professional logo builder, marketing automations with WooCommerce and social management -- CreativeMail is a whole lot more than mail from Constant Contact.',
+					'selected' => false,
+					'shown'    => true,
+				),
+				'yith-woocommerce-ajax-search'       => array(
+					'slug'     => 'yith-woocommerce-ajax-search',
+					'icon'     => '--site-features-search',
+					'title'    => 'Enhanced Product Search',
+					'subtitle' => 'Powered by YITH',
+					'desc'     => 'Give your visitors great search experiences with this exclusive offering from our colleagues at YITH.',
+					'selected' => false,
+					'shown'    => true,
+				),
+				'nfd_slug_yith_woocommerce_ajax_product_filter' => array(
+					'slug'     => 'nfd_slug_yith_woocommerce_ajax_product_filter',
+					'icon'     => '--site-features-filter',
+					'title'    => 'Enhanced Product Filters',
+					'subtitle' => 'Powered by YITH',
+					'desc'     => 'Give your visitors powerful tools to discover your great products with this exclusive offering from our colleagues at YITH.',
+					'selected' => false,
+					'shown'    => true,
+				),
+				'nfd_slug_yith_woocommerce_booking'  => array(
+					'slug'     => 'nfd_slug_yith_woocommerce_booking',
+					'icon'     => '--site-features-bookingcalendar',
+					'title'    => 'Bookings & Appointments',
+					'subtitle' => 'Powered by YITH',
+					'desc'     => 'Have visitors book meetings and services with you, accepting payment and more using this exclusive offering from our colleagues at YITH.',
+					'selected' => false,
+					'shown'    => true,
+				),
+				'nfd_slug_yith_woocommerce_wishlist' => array(
+					'slug'     => 'nfd_slug_yith_woocommerce_wishlist',
+					'icon'     => '--site-features-wishlist',
+					'title'    => 'Product Wishlists',
+					'subtitle' => 'Powered by YITH',
+					'desc'     => 'Let discerning shoppers curate their selections with a system of favorites using this exclusive offering from our colleagues at YITH.',
+					'selected' => false,
+					'shown'    => true,
+				),
+				'optinmonster'                       => array(
+					'slug'     => 'optinmonster',
+					'icon'     => '--site-features-lead',
+					'title'    => 'Lead Generation',
+					'subtitle' => 'Powered by Optin Monster',
+					'desc'     => 'Connect with website visitors using a proven kit of tools for growth using this offering from Awesome Motive.',
+					'selected' => false,
+					'shown'    => true,
+				),
+			),
+		);
+	}
+
+	public static function mark_initial_plugins() {
```

Comment:
Sure @wpscholar, So we offer a number of plugins the user can install in the SiteFeatures step, and many of them are already queued for install as soon as the user opens onboarding, so this function just checks and marks the already queued plugins so that these are prechecked for the user when he opens SiteFeatures page.

<img width="1224" alt="Screenshot 2022-12-13 at 9 58 59 AM" src="https://user-images.githubusercontent.com/48691514/207226804-a4dd39f9-4461-4456-8e22-05e331d84857.png">
So the checked ones are either installed or will be installed soon, user can uncheck them to uninstall them or even add new plugins that aren't installed by checking those boxes.

Reply:
@officiallygod Let's add a few doc-blocks to these functions and describe what they do.